packages/web-react/src/components/Toast/README.md

```diff
@@ -185,21 +191,51 @@ Alternatively, a custom icon can be used:
 | `success`     | `check-plain` |
 | `warning`     | `warning`     |
 
-### Action Link
+### ToastBar Components
+
+The content of `ToastBar` can be assembled from the following components:
+
+#### ToastBarMessage
+
+`ToastBarMessage` is a component designates for main message in `ToastBar`.
 
-An action link can be added to the ToastBar component:
+Usage example:
 
 ```jsx
-<ToastBar id="my-toast">
-  Message with action
-  <Link href="#" color="inverted" isUnderlined>
-    Action
-  </Link>
+<ToastBar id="my-toast" isOpen={isOpen} onClose={() => setIsOpen(false)} isDismissible>
+  <ToastBarMessage>This is the main toast message.</ToastBarMessage>
+</ToastBar>
+```
+
+#### API
+
+| Name       | Type        | Default | Required | Description                    |
+| ---------- | ----------- | ------- | -------- | ------------------------------ |
+| `children` | `ReactNode` | —       | ✔        | Content of the ToastBarMessage |
```

Comment:
```suggestion
| `children` | `ReactNode` | —       | ✓        | Content of the ToastBarMessage |
```

Please use the "check mark" instead of the "heavy check mark". See https://github.com/lmc-eu/spirit-design-system/pull/1436.

Same in other API tables in this PR.